README.md

```diff
@@ -47,11 +70,59 @@ class MyElement extends LitElement {
 customElements.define('my-element', MyElement);
 ```
 
+Now, use your new element anywhere in HTML:
+
 ```html
   <my-element whales="5">hi</my-element>
 ```
 
+## Developing with LitElement
+
+LitElement, like Polymer 3.0 and lit-html, is distributed as standard JavaScript
+modules. This is so that the modules load natively in browsers and are readable
+by any tool that support standard JavaScript modules, and compilable to any
+other module format like AMD or CommonJS.
+
+The one caveat is that LitElement imports Polymer using Polymer's package name,
+like so:
+
+```js
+import {PropertiesMixin} from '@polymer/polymer/lib/mixins/properties-mixin.js';
+```
+
+Browsers do not yet natively support such imports - they only import modules
+by full URL or absolute or relative path. Yet many tools - like bundlers,
```

Comment:
Can this section be somehow targeted at developers who are explicitly trying to use `LitElement` directly in the browser?  i.e., don't want to scare off people already in the Rollup/WebPack workflow by making this all sound somehow Polymer-specific.

Reply:
Do you have a suggestion? I don't read this as scaring off anyone, just that our old path pattern isn't that well supported.

Reply:
Well, I was thinking something along the lines of, 

> If you're using WebPack, Rollup, or another tool that already handles compiling out package names from import specifiers for development, you can skip this section.

🤷‍♂️